src/lib/forms/PipelineForm.svelte

```diff
@@ -18,6 +19,24 @@
     atf4_type: "",
     accuracy: "",
     is_alternative: false,
+    intervention_budget: {
```

Comment:
This part is redundant with below. The below one is nicer, because it'll handle files we've been making the last few weeks